exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetComplex.java

```diff
@@ -220,4 +221,510 @@ public void testReadRepeatedDecimals() throws Exception {
         .baselineValues(ints, longs, fixedLen, fixedLen)
         .go();
   }
+
+  @Test
```

Comment:
@KazydubB I see only unit tests for Parquet. Please add tests for  other formats in Drill that support maps. For example Avro: https://avro.apache.org/docs/current/spec.html#Maps

Also please adds tests to check that maps are written correctly for Parquet and Avro.